app/controllers/components/course/assessments_component.rb

```diff
@@ -24,6 +28,18 @@ def main_sidebar_items
     end
   end
 
+  def assessment_submissions
+    [
+      {
+        key: :assessments,
```

Comment:
@weiqingtoh This key need to be unique, otherwise it will override each other's settings.